ops/cli.py

```diff
@@ -1,351 +1,309 @@
+            raise RuntimeError(f"Namespace {config.cluster_namespace} not found in {config.cluster_name}")
+        else:
+            shell(f"kubectl config set-context --current --namespace={config.cluster_namespace}")
+            success("Kubernetes configured")
+    except subprocess.TimeoutExpired as timeout:
+        warn("Namespace query took too long--maybe you need to be on non-split VPN?")
+        raise timeout
+
+
+def configure_helm(config: WflInstanceConfig) -> None:
+    """Configure helm to connect to gotc-helm-repo."""
+    info(f"=>  Setting up Helm charts ahead of {config.command}")
+    shell("helm repo add gotc-charts https://broadinstitute.github.io/gotc-helm-repo/")
     shell("helm repo update")
-    success("[✔] Set up Helm charts")
-
-
-def set_up_k8s(project: str, namespace: str, cluster: str, zone: str):
-    """Connect to K8S cluster and set up namespace."""
-    info(f"=> Setting up Kubernetes with namespace {namespace}.")
-    GCLOUD = " ".join(["gcloud container clusters get-credentials",
-                       f"{cluster} --zone {zone}",
-                       f"--project {project}"])
-    shell(GCLOUD)
-    info("=> Setting up Kubernetes cluster context.")
-    ctx = f"gke_{project}_{zone}_{cluster}"
-    shell(f"kubectl config use-context {ctx} --namespace={namespace}")
-    success(f"[✔] Set context to {ctx}.")
-    success(f"[✔] Set namespace to {namespace}.")
-
-
-def run_cloudsql_proxy(project: str, cloudsql_instance_name):
-    """Connect to a google cloud sql instance using the cloud sql proxy."""
-    info("=> Running cloud_sql_proxy")
-    token = subprocess.check_output("gcloud auth print-access-token", shell=True, encoding='utf-8').strip()
-    instance_command = " ".join([f"gcloud --format=json sql --project {project}",
-                             f"instances describe {cloudsql_instance_name}",
-                             "| jq .connectionName | tr -d '\"'"])
-    instance = subprocess.check_output(instance_command, shell=True, encoding='utf-8').strip()
-    docker_command = " ".join(['docker run --rm -d -p 127.0.0.1:5432:5432 gcr.io/cloudsql-docker/gce-proxy:1.16 /cloud_sql_proxy',
-                        f'-token="{token}" -instances="{instance}=tcp:0.0.0.0:5432"'])
-    return subprocess.check_output(docker_command, shell=True, encoding='utf-8').strip()
-
-
-def run_liquibase_migration(db_username, db_password):
-    """Run liquibase migration on the database that the cloudsql proxy is connected to."""
-    info("=> Running liquibase")
+    success("Set up Helm charts")
+
+
+def render_values_file(config: WflInstanceConfig) -> None:
+    """Render the values file and store pertinent info from it in the config."""
+    info("=>  Rendering Helm values file")
+    deploy = os.path.join("derived", "helm", "deploy")
+    if not os.path.exists(deploy):
+        os.makedirs(deploy)
+    values = "wfl-values.yaml"
+    config.rendered_values_file = os.path.join(deploy, values)
```

Comment:
We store the path in the config so that there's no longer a tight coupling between this function and the helm deploy (this means that the deploy --dry-run can actually make the values file)